gapic/ads-templates/tests/unit/gapic/%name_%version/%sub/test_%service.py.j2

```diff
@@ -1109,7 +1109,7 @@ def test_{{ method_name }}_rest_required_fields(request_type={{ method.input.ide
         ))
 
     # verify fields with default values are dropped
-    {% for req_field in method.input.required_fields if req_field.name in method.query_params %}
+    {% for req_field in method.input.required_fields if req_field.is_primitive and req_field.name in method.query_params %}
```

Comment:
This drops coverage when `is_primitive` is false. How would you test it?

Reply:
This is a rollback. It gets tested the same way as before. This affects the py_test taregets at minimum (the whole reason why I'm changing this is because it fails the generted unit tests)